### ISSUE TITLE
Can't start etherpad on Debian 10

### ISSUE THREAD

**Issue by MalteKiefer** [ether/etherpad-lite#5053]
Hello,

I tried to install and run etherpad on my server.
So this is what I do:

1. Create locale user: `useradd -m -s /bin/bash etherpad`
2. Login as User: `su - etherpad`
3. Run this command: `git clone --branch master https://github.com/ether/etherpad-lite.git && cd etherpad-lite && src/bin/run.sh`

```bash
etherpad@server:~$ git clone --branch master https://github.com/ether/etherpad-lite.git && cd etherpad-lite && src/bin/run.sh 
Ensure that all dependencies are up to date...  If this is the first time you have run Etherpad please be patient.

added 682 packages, and audited 1112 packages in 8s

2 packages are looking for funding
  run `npm fund` for details

10 vulnerabilities (1 low, 4 moderate, 3 high, 2 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues, run:
  npm audit fix --force

Run `npm audit` for details.
Clearing minified cache...
Starting Etherpad...
[2021-06-06 21:47:07.674] [DEBUG] console - Running on Node v16.3.0 (minimum required Node version: 10.17.0)
[2021-06-06 21:47:07.694] [INFO] console - All relative paths will be interpreted relative to the identified Etherpad base dir: /home/etherpad/etherpad-lite
[2021-06-06 21:47:07.696] [DEBUG] AbsolutePaths - Relative path "settings.json" can be rewritten to "/home/etherpad/etherpad-lite/settings.json"
[2021-06-06 21:47:07.696] [DEBUG] AbsolutePaths - Relative path "credentials.json" can be rewritten to "/home/etherpad/etherpad-lite/credentials.json"
[2021-06-06 21:47:07.708] [INFO] console - settings loaded from: /home/etherpad/etherpad-lite/settings.json
[2021-06-06 21:47:07.709] [INFO] console - No credentials file found in /home/etherpad/etherpad-lite/credentials.json. Ignoring.
[2021-06-06 21:47:07.710] [WARN] console - Unknown Setting: 'dumpOnUncleanExit'. This setting doesn't exist or it was removed
[2021-06-06 21:47:07.711] [INFO] console - Using skin "colibris" in dir: /home/etherpad/etherpad-lite/src/static/skins/colibris
[2021-06-06 21:47:07.711] [INFO] console - Session key loaded from: /home/etherpad/etherpad-lite/SESSIONKEY.txt
[2021-06-06 21:47:07.712] [INFO] console - Random string used for versioning assets: 2458d08b
[2021-06-06 21:47:07.980] [INFO] server - Starting Etherpad...
node[13632]: ../src/api/callback.cc:129:void node::InternalCallbackScope::Close(): Assertion `(env_->execution_async_id()) == (0)' failed.
 1: 0xb17ec0 node::Abort() [node]
 2: 0xb17f3e  [node]
 3: 0xa59021 node::InternalCallbackScope::Close() [node]
 4: 0xa590e1 node::InternalCallbackScope::~InternalCallbackScope() [node]
 5: 0xa90a24 node::Environment::RunTimers(uv_timer_s*) [node]
 6: 0x158b495  [node]
 7: 0x158f9f5 uv_run [node]
 8: 0xa59f25 node::SpinEventLoop(node::Environment*) [node]
 9: 0xb59bd7 node::NodeMainInstance::Run(node::EnvSerializeInfo const*) [node]
10: 0xadfaa2 node::Start(int, char**) [node]
11: 0x7f87097ee09b __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
12: 0xa57d6c  [node]
Abgebrochen
etherpad@server:~/etherpad-lite$ 
```

**Server**
* OS: Debian 10 Buster
* Node: v16.3.0
* NPM: 7.15.1
* Etherpad: latest

**Comment by rhansen** [comment]
More information is needed. Please follow the [new bug instructions](https://github.com/ether/etherpad-lite/blob/develop/.github/ISSUE_TEMPLATE/bug_report.md).

**Comment by MalteKiefer** [comment]
I updated the issue

**Comment by webzwo0i** [comment]
Thanks for the report. I believe the bug is fixed in a001a13411e377773b1e17ebda04585cb4b05070. It should be save to cherry-pick in case you want to give it a try

**Comment by webzwo0i** [comment]
In case your problem is not fixed by this commit, please feel free to reopen this issue :-)
